Source/JavaScriptCore/inspector/protocol/Canvas.json

```diff
@@ -23,7 +23,7 @@
         {
             "id": "ContextType",
             "type": "string",
-            "enum": ["canvas-2d", "bitmaprenderer", "webgl", "webgl2"],
+            "enum": ["canvas-2d", "offscreen-canvas-2d", "bitmaprenderer", "webgl", "webgl2"],
```

Comment:
Not sure about this one yet, I'm not that familiar with frontend/backend of inspector. Maybe a future refactoring?

Reply:
i suppose since there's a different API surface it would be better to do it this way.  it would also make it easier for `Recording` since we wouldn't need another thing there (e.g. `Recording.Source` or `Recording.Creator` or etc.)